packages/cli/src/cli/services/upgrade.ts

```diff
@@ -77,26 +77,15 @@ async function upgradeGlobalShopify(currentVersion: string): Promise<string | vo
 
   outputUpgradeMessage(currentVersion, newestVersion)
 
-  const homebrewPackage = process.env.SHOPIFY_HOMEBREW_FORMULA as HomebrewPackageName | undefined
   try {
-    await (homebrewPackage ? upgradeGlobalViaHomebrew(homebrewPackage) : upgradeGlobalViaNpm())
+    await upgradeGlobalViaNpm()
   } catch (err) {
```

Comment:
if we detect this is a homebrewPackage, shouldn't we cancel and warn the user to use brew?
Otherwise it will try to upgrade the global shopify using npm

Reply:
Yep! I'm changing the code to have that behaviour instead.